compiler/luci/service/src/Nodes/CircleConv2D.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <luci/Service/CircleShapeSignatureInference.h>
+
+namespace luci
+{
+
+ShapeSignature ssinf::Algorithm::visit(const luci::CircleConv2D *node)
+{
+  return signature_of_arg(node, 0);
```

Comment:
I don't think this will work for `CircleConv2D `
Please refer `infer_conv2d_type()` in `compiler/luci/service/src/CircleShapeInferenceRule.cpp` 
If you need information about how output shape of convolution is, please search with `convolution output shape`